tests/10-vcdm2.js

```diff
@@ -294,6 +321,7 @@ describe('Verifiable Credentials Data Model v2.0', function() {
         await assert.rejects(endpoints.issue(require(
           './input/credential-issuer-object-id-not-url-fail.json')));
       });
+      // TODO: remove. no longer present.
```

Comment:
the text of the normative statement just changed: https://w3c.github.io/vc-data-model/#names-and-descriptions

Reply:
Thanks. Noted.